crowbar_framework/spec/models/api/crowbar_spec.rb

```diff
@@ -125,6 +125,21 @@
     end
   end
 
+  context "with ceph cluster healthy" do
+    it "succeeds" do
+      expect(subject.ceph_healthy?).to be true
```

Comment:
instead of relying, that the test environment doesn't have ceph, i would test it like this:

```ruby
it "succeeds to check ceph cluster health" do
  allow(Node_Object).to receive(:find).with(
    "roles:ceph-mon AND ceph_config_environment:*"
  ).and_return([NodeObject.find_node_by_name("testing.crowbar.com")])
  allow_any_instance_of(NodeObject).to receive(:run_ssh_cmd).with(
    "ceph health 2>&1"
  ).and_return(exit_code: 0, stdout: "Some message", stderr: "")

  expect(subject.ceph_healthy?).to be true
end
```